README.md

```diff
@@ -41,74 +40,85 @@ brew install cmake git wget curl bash-completion qemu
 python3 -m pip install --user -U  colcon-common-extensions rosdep vcstool
 ```
 
-[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/)
+[Install Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_mac/) on Mac.
+
+
+### Installing from source
+
+#### Latest (unstable development - `master` branch)
+Please follow those instructions if you plan to contribute to this repository.
+
+* Install all software dependencies required for ROS 2 development by following the [ROS 2 documentation](https://index.ros.org/doc/ros2/Installation/Latest-Development-Setup/)
+* Checkout the source code and compile it as follows
+
+```bash
+cd my_ros2_ws/src
+git clone https://github.com/ros-tooling/cross_compile
+
+# Build the source packages
+cd my_ros2_ws
+
+# If this is the first time building the workspace
+colcon build
```

Comment:
remove please, this is unnecessary and will hide bugs if the packages is missing dependencies.

Reply:
please align:

```
curl https://raw.githubusercontent.com/aws-robotics/launch-ros-sandbox/master/cross_compile.repos | vcs import src/
# Install all required system dependencies
rosdep update
rosdep install --ignore-packages-from-source --from-paths src/
# Use colcon to compile launch_ros_sandbox code and all its dependencies
colcon build --packages-up-to cross_compile
```

please align with launch_ros_sandbox

Reply:
Yes, done in the latest commit.